test/notifier/message_test.rb

```diff
@@ -0,0 +1,93 @@
+require "minitest/autorun"
+require "mocha/minitest"
+
+require "serverless-tools/notifier/message"
+
+module ServerlessTools::Notifier
+  describe Message do
+    let(:mock_git_client) { mock("git_client") }
+    let(:repo_name) { "fac/repo-name" }
+    let(:run_id) { 123 }
+    let(:sha) { "dab326e948974caba97eae82a1431d0bfcdeff36" }
+    let(:deploy_info) do
+      "<https://github.com/fac/repo-name/actions/runs/3534407323/attempts/1|fac/repo-name/branch-name #643> " \
+      "for @terry.pratchett\n" \
```

Comment:
I always like to see which names other use for testing. Personally, I go for Lost characters.